minikube.mk

```diff
@@ -1,7 +1,9 @@
+ARGO_WORKFLOWS_CHART_VERSION=0.17.0
+ARGO_EVENTS_CHART_VERSION=2.4.1
```

Comment:
In `integration-test-up` we use a lengthy `sed` command to grab the argo workflows version from `go.mod`. This doesn't work here because we're installing argo workflows using the helm chart, so we need the helm chart version (same goes for argo events).

There is a workaround (haven't tried it) but it's a bit hacky: we can override the image versions in the helm values to point to the correct version of argo workflows (same with events). It would look something like this:

```
$(HELM) install argo-workflows argo/argo-workflows -n argo --create-namespace \
  --set "argo-workflows.controller.image.tag=3.4.5" \
  --set "argo-workflows.server.image.tag=3.4.5"
```